docs/privacy-policy.md

```diff
@@ -0,0 +1,81 @@
+# Privacy Policy
+
+#### _Effective:_ February 8, 2022
+
+
+This policy document explains how the Operate First Data Science Community Meetup (“we” / “us” / “our”) collects, stores, and uses attendees’ personal information. If you wish to access any particular section of this Privacy Policy, use the list below to jump to that section.
+
+1. [Collection Of Information](#Collection-Of-Information)
+
+2. [Use Of Information](#Use-Of-Information)
+
+3. [Sharing Of Information](#Sharing-Of-Information)
+
+4. [Additional Details](#Additional-Details)
+
+5. [References](#References)
+
+6. [Contact Us](#Contact-Us)
+
+
+
+## Collection Of Information
+
+We use Google Meet to host the meetups. Through Google Meet, we get the following information about you, when you attend when you attend the Operate First Data Science Community Meetups:
+1. Your name
+
+    *a.* as it appears on your Google Account, in case you join through your Google Account; or
+
+    *b.* as you provide in the “Your name” section, in case you join anonymously
+
+2. Email ID or phone number you used to join the event
+3. Timestamp(s) when you joined the event
+4. Timestamp(s) when you left the event
+
+We use Google Analytics on the operate-first website. This means that, through Google Analytics, we get the following information about you, when you visit our website:
+Where you got to the page from (e.g. mail,github, twitter, youtube, etc.)
+We do **not** get any personal information through this.
+
+We may also collect information about you such as your name and email ID, when you communicate with us via email or slack.
+
+In addition to this, we also collect visitor metrics and related information like where did you get to know about this page, whenever you visit the website.
+
+## Use Of Information
+
+We use information about you to:
+
+1. Promote our Community;
+2. Track engagement;
+3. Communicate with you; and
+4. Improve content.
+
+## Sharing Of Information
+
+We do not share information about you except as described in this Privacy Policy. We may disclose personal information about you, including:
```

Comment:
Just a thought on making it a bit concise. Lmk if it makes sense.
```suggestion
We do not share information about you except as described in this Privacy Policy below,
```

Reply:
I think it's vital to mention that we might be using this personal information for various reasons as mentioned here.